lib/chef/azure/service.rb

```diff
@@ -6,9 +6,48 @@ class ChefService
   include ChefAzure::Shared
   AZURE_CHEF_SERVICE_PID_FILE = "azure-chef-client.pid"
   AZURE_CHEF_CRON_NAME = 'azure_chef_extension'
+  DEFAULT_CHEF_SERVICE_INTERVAL = 30
+  CLIENT_RB_INTERVAL_ATTRIBUTE_NAME = 'interval'
+

```

Comment:
@mwrock I did not understand this completely. But what I see is that a lot of the methods of `ChefService` class are called from `private` methods of other class like `EnableChef`. I will check which all methods of `ChefService` class can be made `private`.